csun-metalab/nomi
# NAM-434 -- UI: Gallery view - card grid breaks when student name carries to new line

# Description
Implemented a new layout for gallery view and added more responsiveness.
  
# Testing
1. Ensure gallery view doesn't break in different screen resolutions.
2. Check to see if cards will grow in height when name wraps onto the next line.
3. Ensure button is attached at the proper position at all screen resolutions.
